ui/webui/test/README.rst

```diff
@@ -243,3 +266,60 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI integration tests
+========================
+
+Integration (or end to end) tests are stored in ``./ui/webui/test/integration``
+together with tplib test cases and test plans (those are only needed when running
+the tests using Permian).
+
+Integration tests use one more level of abstraction, to make it easier to write new
+tests and avoid code duplication class `IntegrationTest <https://github.com/rhinstaller/anaconda/blob/9baed94ac63fb2da73231ccc27ed88ac4360858b/ui/webui/test/helpers/integration.py#L38>`_
+should be used. This class handles the process of going through all the required
+installation steps with default options. So when writing new test you only have
+to use this class as parent and reimplement the functions that are important for
+the test case.
+
+Integration tests examples
+--------------------------
+
+There are three test cases in the anaconda repository that can be used as examples.
+
+**Default**
+
+Performs basically default installation.
```

Comment:
'basicaly' is a filler word here, and therefore redundant.

Reply:
Removed